### DISCUSSION TITLE
Traspose Rangename with Styles

### DISCUSSION THREAD

**Discussion by ssada1975** [PHPOffice/PhpSpreadsheet#3541] [General]
This is:

```
- [ ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();

// add code that show the issue here...
```
If this is an issue with reading a specific spreadsheet file, then it may be appropriate to provide a sample file that demonstrates the problem; but please keep it as small as possible, and sanitize any confidential information before uploading.

### What features do you think are causing the issue

- [ ] Reader
- [ ] Writer
- [ ] Styles
- [ ] Data Validations
- [ ] Formula Calculations
- [ ] Charts
- [ ] AutoFilter
- [ ] Form Elements

**Comment by ssada1975** [comment]
How to transpose a rangename with styles?

**Comment by MarkBaker** [comment]
I'm not clear what you're asking.
How would you do what you want to do in MS Excel?

**Reply by ssada1975** [reply]
I have a range of data with styles.
I would like to convert rows in columns maintaining the style of cells
